src/ansys/fluent/core/solver/flobject.py

```diff
@@ -33,10 +33,10 @@
 
 
 class InactiveObjectError(RuntimeError):
-    """Provides the error when the object is inactive."""
+    """Inactive object access.."""
```

Comment:
Looks like a bonus .
```suggestion
    """Inactive object access."""
```